src/main/java/core/basesyntax/service/strategy/PurchaseHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+
+public class PurchaseHandler implements OperationHandler {
+    @Override
+    public void getOperation(FruitDao fruitDao, String fruit, int quantity) {
+        fruitDao.addCsvRow(new Fruit(Fruit.Operation.PURCHASE, fruit, quantity));
+    }
```

Comment:
There is no point to have different handlers that contain identical logic.
instead of call someHandler.getOperation(fruitDao, fruit, quantity) you can esaily call fruitDao.addCsvRow(...)

please try to delegate some logic to handlers that will be different.
For example calculation of quantity of some fruit depending on operation. As in one case it would be subtraction, in other addition, in another replacement.
subtraction and replacement also could have some validations not to produce negative balance of fruit after operation performed